app/src/main/java/com/example/smsrc/signin/presenter/SigninPresenter.java

```
@@ -29,11 +30,10 @@ public boolean signUpUser(String username,
                 userRepository.insert(username, Crypto.encrypt(password) , AuthRoles.OWNER);
                 authenticate.authenticate(username, password);
             } catch (Exception e) {
+                Log.e("SigninPresenter", e.getMessage());
                 throw e;
             }
```

Can you please remove this try-catch and let it fail